src/tokens/tokens.interfaces.ts

```diff
@@ -56,21 +95,23 @@ export class TokenPool {
   @IsNotEmpty()
   symbol: string;
 
+  @ApiProperty()
+  @IsNotEmpty()
+  type: TokenType;
+
   @ApiProperty({ description: contractConfigDescription })
   @IsOptional()
-  config?: any;
+  config: {
+    address: string;
+  };
 
   @ApiProperty()
   @IsOptional()
-  data?: string;
+  data: any;
```

Comment:
I think FireFly guarantees this will be a string or undefined. Is there a reason to use `any`?